src/attr/_funcs.py

```diff
@@ -32,6 +33,9 @@ def asdict(
     :param bool retain_collection_types: Do not convert to ``list`` when
         encountering an attribute whose type is ``tuple`` or ``set``.  Only
         meaningful if ``recurse`` is ``True``.
+    :param callable value_serializer: A hook that is called for every attribute
+        or dict key/value.  It receives the current instance, field and value
+        and must return the (updated) value.
```

Comment:
```suggestion
    :param Optional[callable] value_serializer: A hook that is called for every
        attribute or dict key/value.  It receives the current instance, field and
        value and must return the (updated) value.
```

(I'm eyeballing the line length here, I hope it works out 🤪)

Reply:
Narrator: It didn’t.